### PR TITLE
CLI-138: Migrate to v3; add svc env to flex namespace

### PR DESCRIPTION
**Pull request by DennyGD**
Migrates the CLI to v3 API version. Option to specify the service environment is added to the flex namespace: ~~svcEnv~~ env.
Should be merged only when v3 is released.

### CODE REVIEW

File: lib/Utils.js [hunk 1]
```diff
@@ -192,8 +192,9 @@ Utils.Endpoints = {
   orgs: (schemaVersion, orgId) => `${Utils.Endpoints.version(schemaVersion)}/organizations${Utils.Endpoints.getIdPartFromId(orgId)}`,
   servicesByDomain: (domainType, domainId, serviceId, schemaVersion) => `${Utils.Endpoints.version(schemaVersion)}/${domainType}/${domainId}/data-links`,
```
Comment (dokov) [inline lib/Utils.js]: There is still data-links in the URL here. Is this something that is no longer used or is it "data-links" in the new API?

Comment (DennyGD) [reply lib/Utils.js]: Yes, removed.

File: __snapshots__/flex-status.test.js [hunk 2]
```diff
@@ -1,9 +1,31 @@
-exports['flex status by specifying a profile and existent serviceId should succeed and output JSON 1'] = `
+exports['flex status by specifying a profile existent serviceId and existent svc ID should succeed and output default format 1'] = `
```
[the remark this answers is not in the document]
Comment (DennyGD) [reply __snapshots__/flex-status.test.js]: Well, if you execute the command by specifying a profile (--profile), a service ID and a service environment ID that exist (are in the db), then the command should provide the requested data in the human-friendly format (which is the default).

Even though, 'existing' might be better suited here, I think that 'existent' is fine, too. Also, it is used throughout the tests and it is not worth the effort.
https://www.merriam-webster.com/dictionary/existent
https://dictionary.cambridge.org/dictionary/english/existent

[the remark this answers is not in the document]
Comment (dokov) [reply __snapshots__/flex-status.test.js]: So, the second, "svc ID" actually means "service environment ID"?

[the remark this answers is not in the document]
Comment (DennyGD) [reply __snapshots__/flex-status.test.js]: Yes, it is fixed now

**Comment by DennyGD** [comment]
I had to update the branch again. @dokov you might have to take a look 😃

**Pull request merged**